src/app/weather-forecast/weather-forecast.component.html

```diff
@@ -0,0 +1 @@
+<div id="weatherForecastSpace"></div>
```

Comment:
No elements with `id` the component should be used multiple time without any problem.